include/odp/api/spec/ipsec.h

```diff
@@ -832,15 +832,16 @@ odp_ipsec_sa_t odp_ipsec_sa_create(const odp_ipsec_sa_param_t *param);
  * the SA and be processed successfully.
  *
  * When in synchronous operation mode, the call will return when it's possible
- * to destroy the SA. In asynchronous mode, the same is indicated by an
+ * to destroy the SA. In asynchronous mode, the same may be indicated by an
```

Comment:
It seems your intent here is to indicate that in async mode the disable may be completed synchronously as well as asynchronously while in synchronous mode it always completes synchronously. If that's the case this should be stated explicitly rather than being inferred.